hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/ImmutableMemStoreLAB.java

```diff
@@ -138,5 +138,16 @@ public boolean isOffHeap() {
     return ChunkCreator.getInstance().isOffheap();
   }
 
+  /**
+   * This method is only used for test.
+   */
+  public int getRefCntValue() {
+    return this.refCnt.refCnt();
+  }
+
+  boolean isClosed() {
```

Comment:
Only for test too?

Reply:
yes, already used RestrictedApi to fix the problem.